andela/mag-ah-frontend
# #161646092 Add floating create and edit buttons

#### What does this PR do?
Enable user to create article from the root page which lists all articles
Enable user to edit an article from the single article page

#### Description of Task to be completed?
Add floating button on all articles page to create an article. Only appears if a user is authenticated
Add a floating button on a single article page to edit an article. Only appears if a user is authenticated and the author of the article.

#### How should this be manually tested?
Clone: `https://github.com/andela/mag-ah-frontend.git`
Checkout to: `ch-floating-buttons-161646092 `
Install requirements: `npm install`
Create env file: `mv .envsample .env`
Run app: `npm start`
Login into the application

Navigate to the following url:

`http://localhost:3000/articles` to view the floating button.
`http://localhost:3000/{slug}` where the article is authored by you. You should see an edit button

#### What are the relevant pivotal tracker stories?
[Users should be able to view and edit their profiles](https://www.pivotaltracker.com/story/show/161646092)

#### Screenshots

`Create article`

<img width="1333" alt="screen shot 2018-11-01 at 14 35 40" src="https://user-images.githubusercontent.com/12625365/47849650-872d2e80-dde3-11e8-9ed4-2ed07397b7ca.png">


`Update article`

<img width="1322" alt="screen shot 2018-11-01 at 14 35 55" src="https://user-images.githubusercontent.com/12625365/47849657-8f856980-dde3-11e8-94c2-be095efe8665.png">
